alchemy/src/cloudflare/worker.ts

```diff
@@ -179,6 +179,13 @@ export interface BaseWorkerProps<
     enabled?: boolean;
   };
 
+  /**
+   * Enable Workers LogPush for trace event export.
+   * Requires separate LogPush job configuration via Cloudflare API.
+   * @default false
+   */
+  logpush?: boolean;
```

Comment:
This isn't very clear. Should we link to some official docs?

Reply:
Improved the comment slightly and reference the CF docs now